csharp/Platform.IO/ConsoleCancellation.cs

```diff
@@ -54,6 +82,15 @@ public void Wait()
             }
         }
 
+        /// <summary>
+        /// <para>Removes the <see cref="OnCancelKeyPress"/> handler, for the <see cref="Console.CancelKeyPress"/> event and attempts to dispose the object.</para>
```

Comment:
OnCancelKeyPress is private, it is implementation details.
`Unsubscribes from the <see cref="Console.CancelKeyPress"/> event and attempts to dispose the <see cref="CancellationTokenSource"/>.`